### ISSUE TITLE
Build container from a build context, other than git path

### ISSUE THREAD

**Issue by mohammedzee1000** [CentOS/container-pipeline-service#284]
While exploring containers, i have come whose build context differs from the location of the actual dockerfile itself.

For example: 
Openshift ansible:
https://github.com/openshift/openshift-ansible

Dockerfile location (w.r.t repo root): `images/installer/Dockerfile`

Build context for building the container (w.r.t repo root): /

Docker build : `docker build -f images/installer/Dockerfile -t openshift/openshift-ansible` .

Basically you need to be at top level folder while building the image

We should allow user to specify their own build roots when ever needed

**Comment by navidshaikh** [comment]
@mohammedzee1000 : Does that mean, if `openshift-ansible` is indexed for r.c.o (as its current location), we won't be able to build it ?

**Comment by mohammedzee1000** [comment]
@navidshaikh i believe so as we cd into git path and then essentailly do a docker built -t tag -f targetfile .

**Comment by navidshaikh** [comment]
@bamachrn : ping PTAL

**Comment by bamachrn** [comment]
in current scenario, if the dockerfile can be built with the context of the same location as dockerfile. Then we can build the same in pipeline. but here I see a different scenario not covered in pipeline yet.
@navidshaikh @mohammedzee1000

**Comment by mohammedzee1000** [comment]
I guess we should consider trying to get this done :)

**Comment by bamachrn** [comment]
done with PR: https://github.com/CentOS/container-pipeline-service/pull/365. closing the issue.
